lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -59,8 +88,16 @@ dampen_igp_metric:
 
 dampening:
   auto_default: false
-  config_get_token_append: '/^dampening(?: (?:(\d+) (\d+) (\d+) (\d+)|route-map (.*)))?$/'
-  config_set_append: '<state> dampening <route_map> <decay> <reuse> <suppress> <suppress_max>'
+  cli_nexus:
+    config_get_token_append: '/^dampening(?: (?:(\d+) (\d+) (\d+) (\d+)|route-map (.*)))?$/'
+    config_set_append: '<state> dampening <route_map> <decay> <reuse> <suppress> <suppress_max>'
+  cli_ios_xr:
+    config_set: 'router bgp <asnum>'
+    config_set_append:
+      - ' address-family <afi> <safi>'
+      - ' <state> bgp dampening <decay> <reuse> <suppress> <suppress_max>'
+    config_get: 'show running-config router bgp | in bgp dampening'
```

Comment:
The advantage of using the same config_get across all properties is that we can cache the CLI output and only call the device once. If you have a different config_get for each property, that's a different show command needed to call for each property.